src/components/common/Footer.jsx

```diff
@@ -0,0 +1,81 @@
+      <div className="mx-auto max-w-screen-xl">
+          <div className="md:flex md:justify-between">
+              <div className="mb-6 md:mb-0">
+                    <img src={hindfloIcon} className="w-80" alt="Hindflo Logo" />
+              </div>
+              <div className="grid grid-cols-1 sm:gap-6 sm:grid-cols-2">
+                  {/* <div>
+                      <h2 className="mb-6 text-sm font-semibold text-gray-900 uppercase dark:text-white">Resources</h2>
+                      <ul className="text-gray-600 dark:text-gray-400">
+                          <li className="mb-4">
+                              <a href="https://flowbite.com" className="hover:underline">Flowbite</a>
+                          </li>
+                          <li>
+                              <a href="https://tailwindcss.com/" className="hover:underline">Tailwind CSS</a>
+                          </li>
+                      </ul>
+                  </div> */}
+                  <div className="text-center sm:text-start mb-6 sm-mb-0">
+                      <h2 className="mb-2 sm:mb-3 text-sm font-semibold text-gray-900 uppercase dark:text-white">Navigation</h2>
+                      <ul className="text-gray-600 dark:text-gray-400">
+                          <li className="mb-1">
+                              <a href="/" className={`${pathName === "/" && "text-blue-700 font-bold" } hover:underline`}>Home</a>
+                          </li>
+                          <li className="mb-1">
+                              <a href="/about" className={`${pathName === "/about/" && "text-blue-700 font-bold" } hover:underline`}>About Us</a>
+                          </li>
+                          <li>
+                              <a href="/contact" className={`${pathName === "/contact/" && "text-blue-700 font-bold" } hover:underline`}>Contact Us</a>
+                          </li>
+                      </ul>
+                  </div>
+                  <div>
+                      {/* <h2 className="mb-6 text-sm font-semibold text-gray-900 uppercase dark:text-white flex whitespace-nowrap"><img className="size-6 mr-1" src="/images/address.png" alt="Phone"/> Address ABC - XYZ</h2> */}
+                      <ul className="text-gray-600 dark:text-gray-400">
+                          <li className="mb-4">
+                              <a href="#" className="hover:underline flex"><img className="size-6 mr-2" src={addressIcon} alt="Phone"/> Abcde fghijkl mnop qrstuvw xyz</a>
+                          </li>
+                          <li className="mb-4">
+                              <a href="#" className="hover:underline flex"><img className="size-6 mr-2" src={phoneIcon} alt="Phone"/> +910123456789</a>
+                          </li>
+                          <li>
+                              <a href="#" className="hover:underline flex"><img className="size-6 mr-2" src={emailIcon} alt="Phone"/> hindflo@gmail.com</a>
```

Comment:
on click of email id it should open mail box,  please use appropriate properties

Reply:
done